src/database/schema_metadata.py

```
@@ -1644,9 +1644,10 @@ class TableMetadata(TypedDict):
             {"name": "HassoTime", "type": "TEXT", "description": "発走時刻", "example": "1540", "nullable": False},
             {"name": "Umaban", "type": "INTEGER", "description": "馬番", "example": "1", "nullable": False},
             {"name": "TanOdds", "type": "REAL", "description": "単勝オッズ", "example": "3.5", "nullable": True},
-            {"name": "TanNinki", "type": "INTEGER", "description": "単勝人気順", "example": "1", "nullable": True}
+            {"name": "TanNinki", "type": "INTEGER", "description": "単勝人気順", "example": "1", "nullable": True},
+            {"name": "CollectedAt", "type": "TEXT", "description": "collector保存時刻（UTC ISO-8601）", "example": "2026-05-10T12:34:56.789012+00:00", "nullable": True}
```

Ruff (0.15.12)

[warning] String contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF001)